README.md

```diff
@@ -6,15 +6,12 @@ Content used by the DevX API to enhance clients and tooling. At the moment it ha
 This will also be used by the localization team to add translation files, and by feature teams to modify, add, or update samples.
 
 ## Contributing
-## Adding Sample Queries
-
 ### Pre-requisites:
 1. Download and Install Git to your machine https://git-scm.com/downloads
 1. Download and Install VS Code to your machine https://code.visualstudio.com/download
-
+## Adding Sample Queries
 To add sample queries follow these steps:
-1. - If you have write access to this repo (*microsoft-graph-devx-content*), clone the repo to your local machine.
-   - If you don't have write access or you are a new contributor, fork your copy of the repo to your GitHub.
```

Comment:
Instead of deleting these lines, you can extend the message: 
- If you don't have write access to this repo or you are a new contributor, fork your own copy of this repo by clicking on the **Fork** tab at the top right of this page:
![image](https://user-images.githubusercontent.com/40403681/213385808-b5716824-e76a-4549-9d96-cacfa2a4168d.png)